.github/workflows/changeset-check.yml

```
@@ -0,0 +1,127 @@
+              Please add a changeset by running:
+              \`\`\`
+              cd python
+              pnpm changeset
+              \`\`\`
+              
+              This will help document your changes and ensure proper versioning.`
+            })
+
+      - name: Comment on PR if TypeScript changes missing changeset
+        if: steps.filter.outputs.typescript == 'true' && steps.typescript-check.outputs.hasChangesets == 'false'
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)